### DISCUSSION TITLE
1.0.0

### DISCUSSION THREAD

**Discussion by DCSBL** [DCSBL/home-assistant-flitsmeister#4] [Announcements]
# Initial release 🎉 

Let's see how this goes!

<hr /><em>This discussion was created from the release <a href='https://github.com/DCSBL/home-assistant-flitsmeister/releases/tag/1.0.0'>1.0.0</a>.</em>

**Comment by tjorim** [comment]
Is it possible to change the default time units? E.g. for time driven I think hours make more sense. Top 100 sprint in seconds, like the FM app itself.

**Reply by tjorim** [reply]
There are also some home assistant const for that: https://github.com/home-assistant/core/blob/dev/homeassistant%2Fconst.py#L780.

**Reply by DCSBL** [reply]
Yeah absolutely, top sprint in millisecond does not make sense… 

I can change this, but feel free to propose this in a PR 😉

**Reply by tjorim** [reply]
Maybe you drive a rocket instead of a car, opened #7.
